CLDR-15370 Update Survey Tool Known Issues (#5694)
unicode-org/cldr: docs/site/translation.md

Otherwise, please [file a ticket][], but please review this list first to avoid creating duplicate tickets.

> 1. [CLDR-19455] - Items don't re-appear in the Dashboard missing or provisional category if you change your vote to abstain after originally voting in a way to resolve the issue.

Reviewer: ```suggestion
1. [CLDR-19455] - Items don't re-appear in the Dashboard missing or provisional category if you change your vote to abstain after having originally voting for something.
```
